### ISSUE TITLE
make latexpdf fails if rst files contain gif images when building locally. However, readthedocs.org still generates pdfs. Please help!

### ISSUE THREAD

**Issue by brianjking** [sphinx-doc/sphinx#2852]
When I build locally using `make latexpdf` I get this error:

```
Chapter 3.

! LaTeX Error: Unknown graphics extension: .gif.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.264 ...xincludegraphics{{handler-has-teams}.gif}

?
```

However, if I push the documentation to readthedocs.org PDFs can still be generated -- obviously without the animated gif.

Is there a way to replicate this functionality for building locally or using Aerobatic/Travis-CI/etc so that I can still have the gifs in the HTML docs, while still producing PDF docs?

Thanks!

**Comment by tk0miya** [comment]
because PDFTeX does not support GIF files.

I don't know why readthedocs.org is able to build PDFs.
It might convert them to PNG images before building documents.

**Comment by brianjking** [comment]
@tk0miya In the readthedocs.org build log I see the same type of errors as seen in my [initial report](https://github.com/sphinx-doc/sphinx/issues/2852#issue-171073725) when it encounters a gif. However, instead of the build hanging until it's cancelled by me it simply continues on and completes the build.

When I download the PDF it simply has stripped the gif images out, there isn't a PNG replacement.

Any idea how I can achieve this?

Thanks!

**Comment by tk0miya** [comment]
I understand what readthedocs is doing. They invoke `make latexpdf` command with `LATEXOPTS=-interaction=nonstopmode` option.
Then it ignores minor error.

**Comment by tk0miya** [comment]
Fixed by #4537 
Thank you for reporting!
